pom.xml

```diff
@@ -45,6 +46,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
I suspect it's caused by JUnit Plugin decoupling. BTW it has happened before 1.567 IIRC